plugins/resources-generator/src/main/kotlin/dev/icerock/gradle/MultiplatformResourcesPlugin.kt

```diff
@@ -172,7 +176,7 @@ class MultiplatformResourcesPlugin : Plugin<Project> {
     ) {
         val compilations = targets
             .filterIsInstance<KotlinNativeTarget>()
-            .filter { it.konanTarget.family == Family.IOS }
+            .filter { it.konanTarget.family == Family.IOS || it.konanTarget.family == Family.OSX }
```

Comment:
here we can check `konanTarget.isAppleFamily`

Reply:
Fixed with a7d2379fa580990aa84403880db94e59e4a9e7bb